omero/import_scripts/Populate_Metadata.py

```diff
@@ -136,11 +136,11 @@ def run_script():
     client = scripts.client(
         'Populate_Metadata.py',
         """
-    This script processes a csv file, attached to a container,
+    This script processes a CSV file, attached to a container,
     converting it to an OMERO.table, with one row per Image or Well.
     The table data can then be displayed in the OMERO clients.
-    For full details, see
-    http://help.openmicroscopy.org/scripts.html#metadata
+    For full details of the supported CSV format, see
+    https://github.com/ome/omero-metadata/
```

Comment:
```suggestion
    https://github.com/ome/omero-metadata/blob/master/README.rst
```
Might be better to link directly to the README? Not essential though

Reply:
I think I prefer the shorter URL. It might be more stable too.